bats/tests/helpers/utils.bash

```diff
@@ -58,3 +58,20 @@ try() {
         count=$((count + 1))
     done
 }
+
+update_allowed_patterns() {
```

Comment:
I think the function should stay inside `tests/containers/allowed-images.bats`; I don't see how it is being re-used in other tests. I just wanted the functionality pulled out into a function instead of being repeated multiple times.